portality/crosswalks/journal_form.py

```diff
@@ -430,7 +433,11 @@ def bibjson2form(cls, bibjson, forminfo):
     def admin2form(cls, obj, forminfo):
         forminfo['notes'] = []
         for n in obj.ordered_notes:
-            note_obj = {'note': n['note'], 'note_date': n['date'], 'note_id': n['id']}
+            author_id = n.get('author_id', '')
+            note_obj = {'note': n['note'], 'note_date': n['date'], 'note_id': n['id'],
+                        'note_author': Account.get_name_safe(author_id) or author_id,
```

Comment:
I suggest for `note_author` we add both the name and the user account id, so that the editor can find that user by their id if they need to.  For example

```
'note_author': Account.get_name_safe(author_id) + "(" + author_id + ")"
```

Reply:
fixed with placeholder
![image](https://user-images.githubusercontent.com/97410502/231436857-18eb9715-8766-4c42-9cff-65a823c68a64.png)